vendor/vendor.json

```diff
@@ -900,6 +936,114 @@
 			"revision": "dd168db6051b704a01881df7e003cb7ec9a7a440",
 			"revisionTime": "2016-07-29T07:16:56Z"
 		},
+		{
+			"checksumSHA1": "H81V2Jfwte/WYslbaG9ykrD1+Q8=",
+			"path": "github.com/uber/jaeger-client-go",
+			"revision": "3231c1a9c59534e968cd021da5e283cbd0171397",
```

Comment:
Maybe use a tagged released version like v2.16.0 == 2f47546e3facd43297739439600bcf43f44cce5d here?

Go modules have a preference for released versions, so preferring released versions where possible in vendor.json may ease that transition when the time comes.

Same comment for other deps here that have recent enough releases.